doc/source/command_syntax_usage.rst

```diff
@@ -160,24 +160,23 @@ the 'askpass' box in the Vagrantfile.
 JBoss-EAP and -Scan Variants
 ----------------------------
 
-The JBoss EAP facts are divided into two groups. Rho scans for the
-following facts by default:
-
-  - jboss.eap.running-versions
-
-The following facts are available, but Rho will only scan for them if
-requested using the `--facts` option:
-
-  - jboss.eap.installed-versions
-  - jboss.eap.deploy-dates
-
-The `installed-versions` and `deploy-dates` facts are computed by
-scanning the entire host filesystem with the `find` command. This can
-take a large amount of system resources and could potentially have a
-noticeable impact on the performance of running servers. The default
-facts are a good way to get clues about which systems might have JBoss
-installed before performing a resource-intensive scan, or even
-removing the need for a scan entirely.
+The JBoss facts are divided into two groups. The plain jboss-eap facts
+will look for running EAP processes in the process table, and for EAP
+installations on the filesystem at a short list of common installation
```

Comment:
I know that this suggestion may lead to getting this doc outdated, but would be useful to list which are the common installation locations? Also do we want to have those paths listed?

Reply:
I know it would be nice, but I feel reluctant to make a list like that because it seems like it would inevitably get out of date. What do you think of linking to the path list on GitHub?